Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs

```diff
@@ -178,7 +178,8 @@ public class ZitiIdentity {
 			InitOpts.OnZitiContextEvent += SaveNativeContext;
 			StructWrapper ziti_opts_ptr = new StructWrapper(ziti_opts);
 
-			Native.API.ziti_init_opts(ziti_opts_ptr.Ptr, Loop.nativeUvLoop);
+			int result = Native.API.ziti_init_opts(ziti_opts_ptr.Ptr, Loop.nativeUvLoop);
+			Logger.Info("Result of ziti initialization : " + result);

```

Comment:
remove the logging. instead test the result and throw a runtime exception (pick a type that makes sense) if the init fails